Hunting Queries/SecurityEvent/Suspicious_Login_outside_normal_hours.yaml

```diff
@@ -0,0 +1,107 @@
+id: e7bfbc3f-98c7-4aaa-a64c-de9c058b86b2
+name: Suspicious Login outside normal hours
```

Comment:
Lets title this with Windows so it is clear where it is coming from, also this means a file name change to include Windows.